lib/GADS/Schema/Result/Report.pm

```diff
@@ -272,29 +286,34 @@ Function to create a PDF of the report - it will return a PDF object
 sub create_pdf
 {   my ($self, $record) = @_;
 
-    my $dateformat = GADS::Config->instance->dateformat;
-    my $now        = DateTime->now;
-    $now->set_time_zone('Europe/London');
-    my $now_formatted = $now->format_cldr($dateformat) . " at " . $now->hms;
-    my $updated =
-      $self->created->format_cldr($dateformat) . " at " . $self->created->hms;
-
-    my $config = GADS::Config->instance;
-    my $header = $config && $config->gads && $config->gads->{header};
-    my $pdf    = CtrlO::PDF->new(
-        header => $header,
-        footer => "Downloaded by " . $self->user->value . " on $now_formatted",
-    );
-
-    $pdf->add_page;
-    $pdf->heading( $self->name );
-    $pdf->heading( $self->description, size => 14 ) if $self->description;
-
-    my $fields = [ [ 'Field', 'Value' ] ];
+    my $marking = $self->_read_security_marking;
+    my $logo = $self->instance->site->create_temp_logo;
+
+    my $pdf;
+
+    if($logo) {
+        $pdf    = CtrlO::PDF->new(
+            header => $marking,
+            footer => $marking,
+            logo   => $logo,
+        );
+        $pdf->add_page;
+        my $topmargin = -30;
```

Comment:
Does this require the logo to be a certain size? If so, it might be better retrieving the size of the logo and setting the margin accordingly?

Reply:
Not in my opinion, because the margin-top is a relative mesurement, and I've made it so that the sub-heading is low enough that word-wrap around the logo is no longer a problem (it seemed to be the most efficient way to solve that problem).

Reply:
Okay, great.